recipes-bsp/acpi-tables/files/acpi-tables-load

```diff
@@ -3,8 +3,10 @@
 # You must have configfs enabled for this to work
 
     for TABLE in `ls /kernel/firmware/acpi/ 2>/dev/null`; do
-	dest_dir=$(basename $TABLE .aml)
-	mkdir /sys/kernel/config/acpi/table/$dest_dir
-	cat /kernel/firmware/acpi/$TABLE > /sys/kernel/config/acpi/table/$dest_dir/aml
+        dest_dir=$(basename $TABLE .aml)
+        mkdir /sys/kernel/config/acpi/table/$dest_dir
+        cat /kernel/firmware/acpi/$TABLE > /sys/kernel/config/acpi/table/$dest_dir/aml
     done
 
+    gpioset `gpiofind TRI_STATE_ALL`=0
```

Comment:
This is Edison specific in the generic code. NAK.

Reply:
> This is Edison specific in the generic code. NAK.

I found a way how to do this. Eds gets a specific script, overriding the generic one. When building edison an eds specific package will be built (under edison-poky-linux). When building other machines will be under {arch}-poky-linux. This is behavior as for linux and u-boot.

Reply:
Fine by me.